### PR TITLE
Removed "Thread x started" messages from logging

### PR DESCRIPTION
**Pull request by LKS90**
Changed signature of startLog since it was only used in one place and no information was really lost.

### CODE REVIEW

**Comment by Berdir** [comment]
I don't really see how this is removing the log message I care about (Cron thread 1 started)? That's in SerialLauncher::launchJobs().

**Comment by LKS90** [comment]
This one as well, the message is now really removed and the start time, schedule and duration of every cron job is listed in the config/system/cron/jobs view.

File: src/CronJobListBuilder.php [hunk 1]
```diff
@@ -37,15 +38,22 @@ public function buildHeader() {
    */
   public function buildRow(EntityInterface $entity) {
     /* @var \Drupal\ultimate_cron\Entity\CronJob $entity */
+    $entry = $entity->loadLatestLogEntry();
+    // Start and end as UNIX timestamps.
+    $start_time = $entry->start_time;
```
Comment (Berdir) [inline src/CronJobListBuilder.php]: using temporary variables doesn't seem to help much, we only use those 1-2 times, and using them from the object is only a bit longer...

File: src/CronJobListBuilder.php [hunk 2]
```diff
@@ -37,15 +38,22 @@ public function buildHeader() {
...
+    $end_time = $entry->end_time;
+    // Duration in milliseconds.
+    $duration = ($end_time - $start_time) * 1000;
+    $duration = floor($duration);
```
Comment (Berdir) [inline src/CronJobListBuilder.php]: wondering if that is really a good way to display it, I'd suggest something like (0.01s). Also, displaying it below should use $this->t() with a placeholder.

File: src/Plugin/ultimate_cron/Launcher/SerialLauncher.php [hunk 3]
```diff
@@ -209,28 +209,16 @@ public function launch($job) {
...
-    else {
-      $init_message = t('Launched manually');
-    }
-    $log_entry = $job->startLog($lock_id, $init_message);
-
-    drupal_set_message(t('@name: @init_message', array(
-      '@name' => $job->id(),
-      '@init_message' => $init_message,
-    )));
+    $log_entry = $job->startLog($lock_id);
```
Comment (Berdir) [inline src/Plugin/ultimate_cron/Launcher/SerialLauncher.php]: Still not convinced about touching things like startLog(), those don't end up in the watchdog but a separate table and we afaik create an entry anyway, so it doesn't hurt to keep this.

**Comment by LKS90** [comment]
Made the changes you suggested in the comments.

**Comment by Berdir** [comment]
Looks good now, but some tests whould be nice. not sure if that's easily possible because testing time stuff is always tricky. Merging for now.

**Pull request merged**